yarn.lock

```diff
@@ -499,23 +499,23 @@
     semver "^6.3.0"
     unixify "^1.0.0"
 
-"@hubspot/theme-preview-dev-server@0.0.5":
-  version "0.0.5"
-  resolved "https://registry.yarnpkg.com/@hubspot/theme-preview-dev-server/-/theme-preview-dev-server-0.0.5.tgz#0723eafedb5a040c7fac828f4ff93a0a9eed1667"
-  integrity sha512-y56MKHSyGDwFJJWS3uQ5gmz7/M8EzQUMFASAaBAdj9OySjxb3h6ue0cN5+0ftWimnWisTzw0zwrReKyCj/cj+Q==
+"@hubspot/theme-preview-dev-server@0.0.6":
```

Comment:
Not sure why this one came along for the ride

Reply:
That's actually my bad, i forgot to `yarn install` when i bumped the dep there